acq4/modules/Patch/PatchWindow.py

```diff
@@ -76,7 +77,7 @@ def __init__(self, dm, config):
         #self.statusBar().addPermanentWidget(self.logBtn)
         self.setStatusBar(StatusBar())
 
-        self.stateFile = os.path.join('modules', self.clampName + '_ui.cfg')
```

Comment:
A bunch more in here -- np.os, np.sys, np.np, np.str

Reply:
Mm, fixed. And some added `np.int` and `np.float` additions that can be removed.